empress/xscape/plotcostsAnalytic.py

```diff
@@ -1,103 +1,120 @@
 # plotcostsAnalytic.py
 # Ran Libeskind-Hadas, October 2013
 # Plots the cost space using a matplotlib/pyplot
-# Updated June 2020 to plot costs relative to loss rather than to duplication
 
 import matplotlib
-matplotlib.use('tkagg') # need this so plt.show() works
+from matplotlib.axes import Axes
+
+matplotlib.use('tkagg')  # need this so plt.show() works
 import matplotlib.pyplot as plt
 from shapely.geometry import *
 from .CostVector import *
 from .commonAnalytic import *
 
-def plotcosts(CVlist, transferMin, transferMax, dupMin, dupMax, outfile,
-              log=True, display=False):
-    ''' Plots the cost space for the given CVlist of CostVectors.  The x-axis
-        represents duplication cost (relative to unit cost for loss) and
-        the y-axis represents transfer cost (relative to unit cost for
-        loss).  The x-range is from dupMin to dupMax and the
-        y-range is from transferMin to transferMax.'''
-
+def plot_costs_on_axis(axes: Axes, cost_vectors, transfer_min, transfer_max, dup_min,
```

Comment:
Here you can see that we use transfer_min and transfer_max rather than switch_min and switch_max.  It makes more sense to use "transfer" rather than "switch".  Although they are the same thing, in the DTL model T stands for transfer.

Reply:
Yes.